.github/scripts/build-android.sh

```diff
@@ -24,11 +23,30 @@ if [ $? -ne 0 ]; then
 fi
 
 # Run npm run android in the background
-log "command" "cd android"
-cd android
-log "command" "./gradlew clean" # To prevent "installDebug FAILED" https://stackoverflow.com/a/54955869/12610324
-./gradlew clean
-log "command" "cd .."
-cd ..
-log "command" "npm run android"
-npm run android
+if [ $BUILD_TOOL == 'expo' ]; then
+  log "command" "npm run android"
+  # Run npm run android in the background
+  npm run android &
```

Comment:
Expo, unlike CLI, it starts a watch mode in the initial terminal, so if we don't `kill` it it would running forever in Github Action.